pkg/cloud/ibmcloud/actuators/machine/actuator.go

```diff
@@ -237,7 +261,10 @@ func (ic *IBMCloudClient) Update(ctx context.Context, cluster *clusterv1.Cluster
 		guest.SupplementalCreateObjectOptions.FlavorKeyName == nil ||
```

Comment:
Do we still need those checking? As you already put all of the checking in `validateGuest`?

Reply:
done